resources/fan_control/fan_control.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo /usr/bin/jetson_clocks --fan
```

Comment:
Just a note, sudo might not be required.

Reply:
I get this output on fresh xavier nx installation:
```
dcs_user@dcs:~$ /usr/bin/jetson_clocks --fan
Error: Run this script(/usr/bin/jetson_clocks) as a root user
```

Reply:
As a standalone call from cmd, it indeed needs sudo. But this is being run from the service, which has no  `User` property set. 

Anyway this is not a big issue, the sudo call just might lead to a small warning in the service, but it will work.

Reply:
I cannot post the exact output, but with the change, fan did not start.